Languages/Japanese/DefInjected/ThingDef/Buildings_Furniture_MedPod.xml

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<LanguageData>
+
+  <!-- Standard MedPod -->
+
+  <MedPodStandard.label>治療ポッド</MedPodStandard.label>
+  <MedPodStandard.description>欠損した臓器や手足の再生など、ほぼすべての負傷や疾患の治療に原子レベルの細胞再生技術を利用した高度な医療ベッドです。\n\nこれはGlitterworldでよく見られる無料の公的治療クリニックの標準的な設備です。</MedPodStandard.description>
+  <Blueprint_MedPodStandard.label>治療ポッド (設計)</Blueprint_MedPodStandard.label>
+  <Blueprint_Install_MedPodStandard.label>治療ポッド (移動先)</Blueprint_Install_MedPodStandard.label>
+  <Frame_MedPodStandard.label>治療ポッド (施工)</Frame_MedPodStandard.label>
+  <Frame_MedPodStandard.description>欠損した臓器や手足の再生など、ほぼすべての負傷や疾患の治療に原子レベルの細胞再生技術を利用した高度な医療ベッドです。\n\nこれはGlitterworldでよく見られる無料の公的治療クリニックの標準的な設備です。</Frame_MedPodStandard.description>
+
```

Comment:
You should probably also include translations for the Royalty DLC-only MedPod Lux version. Perhaps it would be called `治療ポッド・ルクス`?

Here are the original English tags, labels and descriptions for you to translate:
```
<!-- EN: MedPod Lux -->
<MedPodLux.label>治療ポッド・ルクス</MedPodLux.label>
	
<!-- An advanced hospital bed that uses atomic cellular regeneration to treat nearly all kinds of injuries and diseases, including regenerating missing organs and limbs.\n\nThis is an luxury model with gold-inlaid fittings and a gel-infused memory foam mattress, capable of faster diagnosis and treatment but requires higher power consumption. -->
<MedPodLux.description>???</MedPodLux.description>
```

Reply:
Oh. MedPod Luxury. ok fixed it.